.codecov.yml

```diff
@@ -0,0 +1,9 @@
+comment: false
```

Comment:
Like @jayvdb mentioned, we don't need a configuration to publish code coverage to codecov.